ui/shared/components/panel/variants/Variants.jsx

```diff
@@ -68,14 +86,76 @@ const Variant = ({ variant }) => {
   )
 }
 
-Variant.propTypes = {
+SingleVariant.propTypes = {
   variant: PropTypes.object,
 }
 
+const CompoundHet = ({ variant }) => {
+  const compoundHetGeneId = getVariantMainGeneId(variant)
+  return (
+    <StyledCompoundHetRow key={variant.variantId} severity={CLINSIG_SEVERITY[(variant.clinvar.clinicalSignificance || '').toLowerCase()]}>
+      <StyledCompoundHetLink width={16}>
+        {variant.familyGuids.map(familyGuid =>
+          <FamilyVariantTags familyGuid={familyGuid} variant={variant} key={variant.variantId} isCompoundHet />,
+        )}
+      </StyledCompoundHetLink>
+      <Grid.Column width={16}>
+        <Pathogenicity variant={variant} />
+      </Grid.Column>
+      <Grid.Column>
+        {Object.keys(variant.transcripts).filter(geneId => geneId !== compoundHetGeneId).map(geneId =>
+          <VariantGene key={geneId} geneId={geneId} variant={variant} compact />,
+        )}
+        {Object.keys(variant.transcripts).length > 1 && <VerticalSpacer height={20} />}
+        {variant.familyGuids.map(familyGuid =>
+          <VariantIndividuals key={familyGuid} familyGuid={familyGuid} variant={variant} />,
+        )}
+      </Grid.Column>
+      <Grid.Column><Annotations variant={variant} /></Grid.Column>
+      <Grid.Column><Predictions variant={variant} /></Grid.Column>
+      <Grid.Column><Frequencies variant={variant} /></Grid.Column>
+      <Grid.Column width={16}>
+        <FamilyVariantReads variant={variant} />
+      </Grid.Column>
+    </StyledCompoundHetRow>
+  )
+}
+
+CompoundHet.propTypes = {
+  variant: PropTypes.object,
+}
+
+const CompoundHets = ({ variants }) => {
+  const geneId = getVariantMainGeneId(variants[0])
```

Comment:
you actually want to use the gene shared between the variants, which isn't always the main gene for the first variant